### PR TITLE
Front end tools

### PR DESCRIPTION
**Pull request by davidtito1992**
## Summary 

Add to the 'index.html' the necessary HTML code to represent the title, and add to the 'application.scss' a new '@import' of a 'titles.scss' file that has the title styles.

## Trello Card

https://trello.com/c/TTvyVRGN/21-page-titles

## Screenshots

captura de pantalla 2018-09-12 a la s 15 42 34

### CODE REVIEW

File: Maquetado/scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  margin: 50px 0 0 100px;
+  padding: 45px 62px 65px;
+  position: relative;
+
+  .text {
+    font-family: "Open Sans", sans-serif;
+    font-size: 25px;
+    position: absolute;
+  }
+
```
Comment (CarlosHWolox) [inline Maquetado/scss/titles.scss]: You could delete this unnecessary space

File: Maquetado/scss/titles.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+$urlImagen: "./assets/triangle.svg";
+
+.front-end{
+  background-image: url($urlImagen);
+  background-repeat: no-repeat;
+  margin: 50px 0 0 100px;
+  padding: 45px 62px 65px;
```
Comment (CarlosHWolox) [inline Maquetado/scss/titles.scss]: One of the most important subjects while coding is to write code for others, not only for readability but to be easy to modify it. Maybe this is not the most clearable way to understand this effect, i can suggest you maybe using the [::before css selector](https://developer.mozilla.org/en-US/docs/Web/CSS/::before) or could it as simple as making an image tag and using layout 'magic' to be more clear with what you want to achieve.

File: Maquetado/index.html [hunk 3]
```diff
@@ -10,10 +10,10 @@
...
-    <p class='text'>Front-End Tools</p>
+  <div class="title">
+    <img class="img">
+    <h2 class="text">Front-End Tools</h2>
   </div>
-
 </body>
 
-</html>
+</html>
```
Comment (CarlosHWolox) [inline Maquetado/index.html]: Enter at the end of file.

File: Maquetado/index.html [hunk 4]
```diff
@@ -10,10 +10,10 @@
 </head>
 
 <body>
-  <div class='front-end'>
-    <p class='text'>Front-End Tools</p>
+  <div class="title">
+    <img class="img">
+    <h2 class="text">Front-End Tools</h2>
```
Comment (CarlosHWolox) [inline Maquetado/index.html]: I would suggest you not so generic class names as it is very probable that you would make more 'title's on the app. Encapsulate them with specific names, like, 'header', 'header-title', 'header-img', 'header-text'.

File: Maquetado/scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    margin: 10px 0 0 10px;
+    padding: 60px;
+    position: absolute;
+  }
+
+  .text {
+    font-family: "Open Sans", sans-serif;
+    font-size: 25px;
+    margin: 60px 0 0 80px;
+    position: absolute;
```
Comment (SKOLZ) [inline Maquetado/scss/titles.scss]: don't use position absolute for both elements, just the image should be absolute positioned, the text should use its space

File: Maquetado/scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+  .img {
+    background-image: url($urlImagen);
+    background-repeat: no-repeat;
+    margin: 10px 0 0 10px;
+    padding: 60px;
+    position: absolute;
+  }
+
+  .text {
```
Comment (SKOLZ) [inline Maquetado/scss/titles.scss]: the h2 should have the class `title` and this properties should be placed there.

**Pull request merged**